require vcs for games and mods (#591)
minetest/contentdb: app/flatpages/policy_and_guidance.md

* **Don't put promotions or advertisements in any package metadata.** <sup>5</sup>
* **Don't manipulate package placement using reviews or downloads.** <sup>6</sup>
* **Screenshots must not be misleading.** <sup>7</sup>
* **Don't submit malware, clearly disclose telemetry, no obfuscated code.** <sup>8</sup>

> * **Use a vcs for mods and games** <sup>9</sup>

Reviewer: How about modpacks?

> * **Use a vcs for mods and games** <sup>9</sup>

Reviewer: ```suggestion
* **Use a VCS (Version Control System) for mods and games** <sup>9</sup>
```

> * **Use a vcs system for mods and games** <sup>9</sup>

Reviewer 2: nit

```suggestion
* **Use a version control system for mods and games.** <sup>9</sup>
```

Author: this is a summary, it is spelled out below, should remove system tho

Author: https://github.com/luanti-org/contentdb/pull/591/commits/eee4476c624cc461b966ebcd33e5cb1cdb3a3d2a

Packages must not contain obfuscated code.

## 9. Version Control System

> For mods, and games a version control system (such as git) is required. Texture packs are currently exempt.

Reviewer 2: ```suggestion
For mods and games a version control system (such as git) is required. Texture packs are currently exempt.
```

## 9. Reporting Violations

Please click "Report" on the package page.

> ## 10. Version Control System

Reviewer 3: Also to be complete, this should be enforced by the code here: https://github.com/luanti-org/contentdb/blob/master/app/logic/package_approval.py#L154

I suppose this could be a follow up PR

> ## 10. Version Control System

Author: solved in https://github.com/luanti-org/contentdb/pull/591/commits/5bbdfde4c1560e1721c35f623b7be4ef344975c5
